llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -1805,6 +1805,9 @@ class VPWidenIntOrFpInductionRecipe : public VPWidenInductionRecipe {
              VPSlotTracker &SlotTracker) const override;
 #endif
 
+  /// Update the step value of the recipe.
+  void setStepValue(VPValue *V) { setOperand(1, V); }
+
```

Comment:
can you move this to `VPWidenInductionRecipe`, after `getStepValue?

Reply:
Done.